index.html

```diff
@@ -11,13 +11,38 @@
         height: 100%;
       }
 
+      #controls {
+        width: inherit;
+        height: 20px;
+        padding:10px;
+
+        display: flex;
+        flex-direction: row;
+      }
+
+      #controls h4 {
+        margin:2px;
+        margin-right: 20px;
+      }
+
       #voyager-embed {
         width: inherit;
         height: inherit;
       }
     </style>
   </head>
   <body>
+    <div id="controls">
```

Comment:
Yeah I could do this (i actually started with a menu item in my first pass. I guess conceptually i've been treating this as a demo app with examples for developers rather than an app designed for end users. The menu approach would involve more code around the core api of getting and restoring state as it requires IPC communication between the renderer and main thread. So i opted for  showing the api itself with as little code as possible.

How do you think about this app overall? Demo/code playground, or app that people would eventually actually use?